docs/FAQ.md

```diff
@@ -125,6 +125,18 @@ for (k, e) in p.walk():
 
 You can then follow the above with `q3.delete_package(pname, registry=reg, top_hash=p.top_hash)`.
 
+## Do I have to login via quilt3 to use the Quilt APIs? How do I push to Quilt from a headless environment like a Docker container?
+
+Configure [AWS CLI credentials](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-configure.html) 
+and `quilt3` will use the same for its API calls. Select among multiple profiles in your shell as follows:
+```bash
+export AWS_PROFILE=your_profile
+```
```

Comment:
Add a note on "Be sure to run `quilt3 logout` if you've previously logged in."

Reply:
Added